R/helpers.R

```diff
@@ -92,6 +98,52 @@ annualStacksToDTx1000 <- function(annualStacks, whNotNA, ...) {
   return(rastersDT)
 }
 
+#' Convert annual raster stack to \code{data.table}
+#'
+#' @param annualStack DESCRIPTION NEEDED
+#' @param whNotNA DESCRIPTION NEEDED
+#' @param ... DESCRIPTION NEEDED
+#'
+#' @return DESCRIPTION NEEDED
+#'
+#' @export
+#' @importFrom data.table as.data.table set
+#' @importFrom LandR asInteger
+#' @importFrom raster stack
+#' @importFrom usefulFuns cbindFromList
+annualStackToDTx1000 <- function(annualStack, whNotNA, ...) {
```

Comment:
What is the purpose of having this and the other almost identically named (and simpler) function `annualStacksToDTx1000` ?

Reply:
yes, this looks like it's doing the same thing as the existing function. and this wersion is using `do.call()` which we are trying to avoid as it's implicated in the slowdown experienced when hitting a `browser()` call in RStudio